Tests/ValetIntegrationTests/ValetIntegrationTests.swift

```diff
@@ -732,7 +732,130 @@ class ValetIntegrationTests: XCTestCase
             XCTAssertEqual(error as? MigrationError, .keyToMigrateInvalid)
         }
     }
-    
+
+    func test_migrateObjectsMatchingCompactMap_successfullyMigratesTransformedValue() throws {
```

Comment:
Should we also add a test that transforms the `key`? (Same for the Valet -> Valet equivalent below)

Reply:
Good call! Will add

Reply:
e990014669e0e8fa072aaef73403d0095497c6b7